packages/connect/src/types/api/cancelCoinjoinAuthorization.ts

```diff
@@ -0,0 +1,10 @@
+import type { PROTO } from '../../constants';
+import type { Params, Response } from '../params';
+
+interface CancelCoinjoinAuthorization {
+    preauthorized?: boolean;
+}
+
+export declare function cancelCoinjoinAuthorization(
+    params: Params<CancelCoinjoinAuthorization>,
```

Comment:
```suggestion
    params?: CommonParams,
```

sicne this method is using only common params there is no need to redeclare them in `CancelCoinjoinAuthorization` - see for example `backupDevice`

Reply:
`preauthorized` is not in `CommonParams` though?

Reply:
you are right, i got confused :+1: